Teacher Workout/Features/Authentication/SignUp/View/SignUpFormView.swift

```diff
@@ -0,0 +1,47 @@
+import SwiftUI
+
+struct SignUpFormView: View {
+    @StateObject var viewModel = SignUpFormViewModel()
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 20) {
+            InputFieldView(label: AppStrings.Authentication.Email.inputLabel.rawValue.localized(),
+                           iconName: "envelope",
```

Comment:
The email address field is capitalised by default. This could lead to authentication issues when we have the server in place.

![Simulator Screen Shot - iPhone 12 mini - 2021-08-01 at 12 11 21](https://user-images.githubusercontent.com/481425/127765812-a9483424-1eeb-403c-bfe5-b95bfd4133e0.png)

Reply:
Are you sure? Autocapitalization is already set to none. I can't reproduce this issue.
<img width="262" alt="Screen Shot 2021-08-01 at 14 38 23" src="https://user-images.githubusercontent.com/15888842/127769471-f8ca46a4-c5ef-4c53-baae-caa64382712b.png">
<img width="302" alt="Screen Shot 2021-08-01 at 14 38 03" src="https://user-images.githubusercontent.com/15888842/127769474-cb50e8eb-d8f3-4d77-94cb-8392e8a7e8c2.png">

Reply:
You are right, is not capitalized. Not sure what happened on my test, probably autocorrect?